contracts/lbp/LBPWrapper.sol

```diff
@@ -44,55 +45,37 @@ contract LBPWrapper {
     /**
      * @dev                       initialize lbp wrapper contract
      * @param _LBPFactory         LBP factory address
-     * @param _swapFeePercentage  swap fee percentage
      */
     function initialize (
-            address _LBPFactory,
-            uint256 _swapFeePercentage
+            address _LBPFactory
     ) public
     {
         owner = msg.sender;
         LBPFactory = ILBPFactory(_LBPFactory);
-        swapFeePercentage = _swapFeePercentage;
-        isInitialized = true;
     }
 
     /**
      * @dev                        initialize lbp wrapper contract
      * @param _name                LBP name
      * @param _symbol              LBP symbol
      * @param _tokens              array of tokens sorted for the LBP
-     * @param _amounts             array of initial amounts used to join pool
      * @param _weights             array of start weights for respective tokens
      * @param _swapEnabledOnStart  enable or disable swap
      * @param _startTime           start time
      * @param _endTime             end time
      * @param _endWeights          array of end weights for respective tokens
-     * @param _admin               address of admin/owner of LBP
-     * @param _userData            extra data required by LBP
      */
     function deployLbpFromFactory(
```

Comment:
let's merge this function and `initialize` into one, and call it `initialize`

Reply:
Maybe call it something else because we're not really only initializing something, we're actually creating something, the LBP. Why not just `deployLbp`?

Reply:
Also we're calling `updateWeightGradually` as well.

Reply:
Let's call it `initializeLBP`